solr/core/src/test/org/apache/solr/cloud/HttpPartitionTest.java

```diff
@@ -70,7 +65,7 @@
  * Simulates HTTP partitions between a leader and replica but the replica does not lose its
  * ZooKeeper connection.
  */
-@LuceneTestCase.Nightly // there are recovery commands that take a while to time out
+// @LuceneTestCase.Nightly // there are recovery commands that take a while to time out
```

Comment:
[Q] Is the commenting-out of the Nightly tag here temporary?  Did you mean to remove this?